config/routes.rb

```diff
@@ -297,15 +302,13 @@
       scope '/other-qualifications' do
         get '/' => 'other_qualifications/type#new', as: :other_qualification_type
         post '/' => 'other_qualifications/type#create'
+        get '/edit/:id' => 'other_qualifications/type#edit', as: :edit_other_qualification_type
+        patch '/edit/:id' => 'other_qualifications/type#update'
```

Comment:
I think `/edit` should always be the last bit of the URL?

```suggestion
        get '/:id/edit' => 'other_qualifications/type#edit', as: :edit_other_qualification_type
        patch '/:id/edit' => 'other_qualifications/type#update'
```

Reply:
So always `/[section-name]/:id/edit` or `/[section-name]/:id/edit/[question]`? That would effect all sections in the candidate interface, which might be a consideration if we were to change this.

Reply:
@paulrobertlloyd If it's one-page per question, I think you could do `/[section-name]/:id/[question]` – no need for the word "edit" at all?